v1.0/openapi3/schemas/Name.yaml

```diff
@@ -3,11 +3,11 @@ type: string
 pattern: '^(?!\s*$)[\p{L}\p{Nd} .,''-]{1,128}$'
 description: |
   The API data type Name is a JSON String, restricted by a regular expression
-  to avoid characters which are generally not used in a name.\n
+  to avoid characters which are generally not used in a name.
   Regular Expression - The regular expression for restricting the
   Name type is \"^(?!\\s*$)[\\w .,'-]{1,128}$\". The restriction does not
   allow a string consisting of whitespace only, all Unicode characters are
   allowed, as well as the period (.) (apostrophe (‘), dash (-), comma (,)
-  and space characters ( ).\n\n\n**Note:** In some programming languages,
+  and space characters ( ).**Note:** In some programming languages,
```

Comment:
Would it be better to put the note on a new line?

Reply:
done